tensorboard/webapp/runs/views/runs_selector/runs_selector_test.ts

```diff
@@ -111,4 +118,104 @@ describe('runs selector test', () => {
       });
     });
   });
+
+  describe('"too many runs" alert', () => {
+    function createRunSelectionMap(runCount: number): Map<string, boolean> {
+      const map = new Map<string, boolean>();
+      for (let i = 0; i < runCount; i++) {
+        map.set(`run${i}`, true);
+      }
+      return map;
+    }
+
+    it('shows when there are too many runs', () => {
+      store.overrideSelector(getExperimentIdsFromRoute, ['123']);
+      store.overrideSelector(
+        getCurrentRouteRunSelection,
+        createRunSelectionMap(MAX_NUM_RUNS_TO_ENABLE_BY_DEFAULT)
+      );
+      const fixture = TestBed.createComponent(RunsSelectorContainer);
+      fixture.detectChanges();
+
+      expect(snackBarOpenSpy).not.toHaveBeenCalled();
+
+      store.overrideSelector(
+        getCurrentRouteRunSelection,
+        createRunSelectionMap(MAX_NUM_RUNS_TO_ENABLE_BY_DEFAULT + 1)
+      );
+      store.refreshState();
+
+      expect(snackBarOpenSpy).toHaveBeenCalledTimes(1);
+    });
+
+    it('does not show when there are too few runs', () => {
```

Comment:
This test is just the first ~10 lines of the previous test, right?  It seems fine to omit (maybe just reword the previous test to something like `it("activates when number of runs exceeds limit")` or something).

Reply:
Done.